tests/markdown/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -1158,19 +1162,21 @@ using Prettier:
 Linters have two categories of rules:
 
 **Formatting rules**: eg: [max-len](http://eslint.org/docs/rules/max-len),
-[no-mixed-spaces-and-tabs](http://eslint.org/docs/rules/no-mixed-spaces-and-tabs),
```

Comment:
These look better before

Reply:
Or should we only break url head if the `[no-mixed-spaces-and-tabs]` is breakable?